src/Miunie.Core/Services/CurrencyService.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace Miunie.Core
+{
+    public class CurrencyService
+    {
+        private const string DailyConversionApiUrl = @"https://www.cnb.cz/cs/financni_trhy/devizovy_trh/kurzy_devizoveho_trhu/denni_kurz.xml";
+        private const string DailyConversionApiUrlDateParam = @"?date=";
+        private const string RowElement = "radek";
+        private const string AmountAttribute = "mnozstvi";
+        private const string CodeAttribute = "kod";
+        private const string CzechCrownsAttribute = "kurz";
```

Comment:
At first I was really wondering what these names refer to, then I realized you're using a Czech Republic site.
All good 😅